File: Day-3/src/main/java/com/springboot/app/Person.java
package com.springboot.app;

public class Person {
    private int age;
    private String name;
    private Address address;


    public Person(int age, String name, Address address) {
        this.age = age;
        this.name = name;
        this.address = address;
        this.test(this);
    }

    public Person(){

    }

    public void test(Person person){

    }

    public int getAge() {
        return age ;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Address getAddress() {
        return address;
    }

    public void setAddress(Address address) {
        this.address = address;
    }
}

File: Day-22/closure/person.js
var age;
var person = (function() {
    var age = 25;
    return {
        age: 100,
        name: "Jack",

        getAge: function() {
            return this.age;
        },

        getOlder: function() {
            age++;
        },

        setAge: function(a) {
            age = a;
        }
    }
}());

console.log(person.name);
console.log(person.getAge());

person.age = 100;
console.log(person.getAge());

person.getOlder();
console.log(person.getAge());
console.log(person);

person.setAge(200);
console.log(person.getAge());